examples/resources/awscc_elasticloadbalancingv2_target_group/elasticloadbalancingv2_target_group_lambda.tf

```diff
@@ -0,0 +1,8 @@
+resource "awscc_elasticloadbalancingv2_target_group" "lambda-example" {
+  name     = "lambda-example"
+  target_type = "lambda"
+  tags = [{
+    key   = "Modified By"
+    value = "AWSCC"
+  }]
+}
```

Comment:
same here, run `terraform fmt` to fix indentation

Reply:
I have rerun `terraform fmt` on the resources file, thanks for noticing.